### PR TITLE
i18n(zh-cn): translate `guides/cms/payload.mdx`

### PR DESCRIPTION
**Pull request by miroim**
#### What kind of changes does this PR include?

- Translated content

#### Description

translate `\guides\cms\payload.mdx`

### CODE REVIEW

**Comment by miroim** [comment]
cc: @liruifengv

**Comment by miroim** [comment]
cc: @liruifengv

File: src/content/docs/zh-cn/guides/cms/payload.mdx [hunk 1]
```diff
@@ -0,0 +1,226 @@
+---
+title: Payload CMS 与 Astro
+description: 使用 Payload CMS 作为 CMS 将內容添加到你的 Astro 项目
+type: cms
+stub: true
+service: Payload CMS
+i18nReady: true
+---
+
+[PayloadCMS](https://payloadcms.com/) 是一个开源无头的 CMS（Content Management System，内容管理系统），可以用来为你的 Astro 项目提供内容。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/payload.mdx:10]:
```suggestion
[PayloadCMS](https://payloadcms.com/) 是一个开源的 headless CMS（Content Management System，内容管理系统），可以用来为你的 Astro 项目提供内容。
```

File: src/content/docs/zh-cn/guides/cms/payload.mdx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+				</li>
+			))
+		}
+	</ul>
+</HomeLayout>
+```
+
+### 使用 PayloadCMS API 生成页面
+
+创建一个页面 `src/pages/post/[slug].astro` 为每一篇文章[动态生成一个页面](/zh-cn/core-concepts/routing/#dynamic-routes)。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/payload.mdx:187]:
```suggestion
创建一个页面 `src/pages/post/[slug].astro` 为每一篇文章[动态生成一个页面](/zh-cn/core-concepts/routing/#动态路由)。
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Pull request merged**